src/scripts/OSFramework/OSUI/Pattern/Search/Search.ts

```diff
@@ -0,0 +1,232 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.Search {
+	/**
+	 * Defines the interface for OutSystemsUI Patterns
+	 */
+	export class Search extends AbstractPattern<SearchConfig> implements ISearch {
+		// Store pattern visibility
+		private _enableNative = false;
+		private _isOpen = false;
+		private _searchGlassButton: HTMLElement;
+		private _searchInput: HTMLInputElement;
+		/**
+		 * Creates an instance of Search.
+		 *
+		 * @param {string} uniqueId
+		 * @param {JSON} configs
+		 * @memberof Search
+		 */
+		constructor(uniqueId: string, configs: JSON) {
+			super(uniqueId, new SearchConfig(configs));
+		}
+
+		private _nativeSearchBehavior(): void {
+			// Store the sate of native behavior
+			this._enableNative = true;
+
+			// Add class for native behavior
+			this.selfElement.classList.add(Enum.CssClass.PatternNative);
+
+			// Define the input selector
+			this._searchInput = Helper.Dom.TagSelector(
+				this.selfElement,
+				OSFramework.OSUI.GlobalEnum.HTMLElement.Input
+			) as HTMLInputElement;
+
+			// Create HTML element for native behavior
+			this._searchGlassButton = document.createElement(OSFramework.OSUI.GlobalEnum.HTMLElement.Div);
+			this._searchGlassButton.classList.add(Enum.CssClass.PatternGlassButton);
+			this.selfElement.appendChild(this._searchGlassButton);
+
+			// Set A11Y properties to elements
+			Helper.A11Y.TabIndexFalse(this._searchInput);
+			Helper.A11Y.TabIndexTrue(this._searchGlassButton);
+			Helper.A11Y.AriaExpandedFalse(this._searchGlassButton);
+			Helper.A11Y.AriaLabel(this._searchGlassButton, 'Open Search');
```

Comment:
Can we add this string into the Enum file?

Reply:
Yes we can. I will change it.